e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -55,6 +55,51 @@ describe('Integrations', () => {
         });
     });
 
+    it('MM-T662 /join command for private channels', () => {
+        const privateChannelName = user1.username;
```

Comment:
nit: Please use name other than the username. Could be,
```suggestion
        const privateChannelName = 'private-channel';
```